doc/mk_sdcard_image.sh

```diff
@@ -71,19 +72,29 @@ partition_image()
 {
 	print_step "Make partitions"
 
-	sudo parted -s $1 mklabel msdos
+	sudo parted -s $1 mklabel msdos || true
```

Comment:
not sure why while partitioning we might want to use skip errors. this does need a comment here

Reply:
Agree. Will remove that.